.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
I don't think its really necessary to run on 4 different node versions

Reply:
What do the various version do?

Reply:
It makes the checks run on each of the different versions of node, to ensure that it will work on all 3. If you're writing some kind of library, that might be important, but for an app, we can control which version of node our code is going to run on, and isn't really necessary

Reply:
Ok, sounds good. Noticed you picked 14.x, why not 15?

Reply:
node LTS (long-term support) is 14.16.x right now until 2023, and 15.x wont ever be active LTS, since node skips every other major version for LTS

Reply:
Thorough. Thanks!